packages/ui/README.md

```diff
@@ -53,6 +53,10 @@ Alternatively you can use `AnchorConnectionProvider` for Anchor Dapps.
 
 `WalletMultiButton` is a component used as the entry point to connect/disconnect a wallet.
 
+| prop               | type     | default |
+| ------------------ | -------- | ------- |
+| maxNumberOfWallets | `number` | `5`     |
```

Comment:
To be consistent with the props value, this should be 3;

Reply:
Oops, must have been looking at my fork in our actual project.